### PR TITLE
misc(build): minify report javascript for lightrider report generator

### PR DESCRIPTION
**Pull request by connorjclark**
cont. #9596 (I meant to just update that PR, but apparently I don't know how to push to a PR branch ...)

I updated @wardpeet's work with master, and also minified the assets for the LR report generator. I verified the report still works with a terser pass (if we rely on function or class names, we'd have to pass special flags, but it seems we don't for the report).

Can confirm this saves a lot of bytes.

image

`dist/lightrider/report-generator-bundle.js` 247 KB -> 152 KB (38%)
`dist/viewer/src/viewer.js` 309 KB -> 213 KB (31%) (done in #9596)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -186,11 +186,15 @@
     },
     {
       "path": "./dist/viewer/src/viewer.js",
-      "maxSize": "76 kB"
+      "maxSize": "65 kB"
```
Comment (connorjclark) [inline package.json]: is 58 now.

File: package.json [hunk 2]
```diff
@@ -186,11 +186,15 @@
...
     },
     {
       "path": "./dist/lighthouse-dt-bundle.js",
       "maxSize": "400 kB"
+    },
+    {
+      "path": "./dist/lightrider/report-generator-bundle.js",
+      "maxSize": "50 kB"
```
Comment (connorjclark) [inline package.json]: is 40 now

File: package.json [hunk 3]
```diff
@@ -94,12 +94,12 @@
     "@types/update-notifier": "^1.0.2",
     "@types/ws": "^4.0.1",
     "@types/yargs": "^8.0.2",
+    "@wardpeet/brfs": "^2.1.0-0",
```
Comment (connorjclark) [inline package.json]: @wardpeet did you rename/republish to wardpack?

**Comment by wardpeet** [comment]
hehe, you should install https://hub.github.com/, it makes it super easy to checkout prs from other branches. Cool that it's getting merged in!

File: package.json (hunk 3 above)
Comment (connorjclark) [reply package.json]: no need, but if you want to ;)

I was mostly making sure you didn't publish to a new package + possible updates

**Comment by connorjclark** [comment]
what, if you consent, can you say the magic phrase: `@googlebot I consent.`

**Comment by wardpeet** [comment]
@googlebot I consent

**Comment by paulirish** [comment]
@googlebot i consent, too.

**Pull request merged**